app/assets/stylesheets/modules.css.scss

```diff
@@ -23,6 +23,20 @@
   }
 }
 
+.resource-block {
+  background:transparent image_url("noun_116372.png") 90% 50% no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Colon after property should be followed by one space